src/components/DatePicker/_DatePicker.scss

```diff
@@ -5,14 +5,22 @@ fieldset {
 
   &.datePicker {
     @include fieldset-reset;
+    margin-bottom: var(--space-m);
   }
 }
 
 .datePicker {
```

Comment:
Why `.datePicker` and not `.date-picker` to follow the formatting of other class names?

Reply:
Yeah, I'll update.

Reply:
Note that some rules like `.datepicker-calendar` come from the React DatePicker package.